### PR TITLE
build(deps): update ol from 6.3.1 to 6.4.2

### PR DESCRIPTION
**Pull request by jacott**

### CODE REVIEW

File: packages/landing/src/map.ts [hunk 1]
```diff
@@ -184,7 +186,7 @@ export class Basemaps {
             .transform(center, `EPSG:${this.config.projection}`, UrlProjection)
             .map((c: number) => Math.floor(c * 10e7) / 10e7); // Limit to 5 decimal places (TODO is 5 enough)
 
-        const zoom = Math.floor(view.getZoom() * 10e3) / 10e3;
+        const zoom = Math.floor(view.getZoom() ?? 0 * 10e3) / 10e3;
```
Comment (blacha) [inline packages/landing/src/map.ts:189]: how does `??` relate to `*`

**Comment by Wentao-Kuang** [comment]
All looks good to me.

File: packages/landing/src/ol.attribution.ts [hunk 2]
```diff
@@ -43,8 +43,9 @@ export class OlAttribution {
     static init(source: Source, map: OlMap, config: MapOptions): void {
         const attribution = new OlAttribution(source, map.getView(), config);
 
-        map.addEventListener(MapEventType.MOVEEND, () => {
+        map.addEventListener(MapEventType.MOVEEND, (): boolean => {
             attribution.updateAttribution();
+            return true;
```
Comment (blacha) [inline packages/landing/src/ol.attribution.ts:48]: what does the return true do here?

File: packages/landing/src/map.ts (hunk 1 above)
Comment (jacott) [reply packages/landing/src/map.ts:189]: badly by the looks of the transpiled code! Well spotted!!

File: packages/landing/src/ol.attribution.ts (hunk 2 above)
Comment (jacott) [reply packages/landing/src/ol.attribution.ts:48]: Continues the event propigation

**Review by blacha — APPROVED** [review]

**Pull request merged**